tests/integration/ha/test_small_deployment_upgrades.py

```diff
@@ -8,62 +8,64 @@
 import pytest
 from pytest_operator.plugin import OpsTest
 
-from .ha.continuous_writes import ContinuousWrites
-from .ha.helpers import app_name, assert_continuous_writes_consistency
-from .ha.test_horizontal_scaling import IDLE_PERIOD
-from .helpers import APP_NAME, MODEL_CONFIG, SERIES, run_action
-from .helpers_deployments import get_application_units, wait_until
-from .tls.test_tls import TLS_CERTIFICATES_APP_NAME
+from ..helpers import APP_NAME, MODEL_CONFIG, SERIES, run_action, set_watermark
+from ..helpers_deployments import get_application_units, wait_until
+from ..tls.test_tls import TLS_CERTIFICATES_APP_NAME
+from .continuous_writes import ContinuousWrites
+from .helpers import app_name, assert_upgrade_to_local
+from .test_horizontal_scaling import IDLE_PERIOD
 
 logger = logging.getLogger(__name__)
 
 
 OPENSEARCH_ORIGINAL_CHARM_NAME = "opensearch"
 OPENSEARCH_CHANNEL = "2/edge"
 
-MACHINE_ID = 0
+
+FIRST_VERSION = "2.12.0"
 
 
 VERSION_TO_REVISION = {
+    FIRST_VERSION: 90,
     "2.13.0": 91,
 }
-FIRST_VERSION = "2.12.0"
-FIRST_REVISION = 90
 
 
-charm = None
+FROM_VERSION_PREFIX = "from_v"
```

Comment:
We should clarify from where to where we are upgrading - it is not immediately clear on the CI when going through the runs. Perhaps; 
```suggestion
GROUP_NAME = "from_v:{}-to_v:local"
```
then use the `.format(version)`  later

Reply:
@Mehdi-Bendriss I cannot use "-" but I will adopt this change.